{
  "name": "test_worst_opcode.py::test_worst_log_opcodes[fork_Prague-benchmark-gas-value_45M-blockchain_test_from_state_test-fixed_offset_False-non_zero_topic-0_bytes_data-log2]",
  "timestamp_completed": "2025-10-02T17:22:13.638586372Z",
  "metadata": {
    "block_used_gas": 45000000
  },
  "execution": {
    "crashed": {
      "reason": "Failed to execute `docker run`: Command \"docker\" \"run\" \"--rm\" \"--env\" \"RUST_LOG=info\" \"--volume\" \"/tmp/.tmpObRKUx:/workspace\" \"ere-cli-openvm:0.0.12-7ef4598\" \"execute\" \"--program-path\" \"/workspace/program\" \"--input-path\" \"/workspace/input\" \"--public-values-path\" \"/workspace/public_values\" \"--report-path\" \"/workspace/report\" failed with status: exit status: 125"
    }
  }
}